libs/langgraph/tests/test_pregel.py

```diff
@@ -9093,6 +9109,704 @@ def parent_2(state: State):
+                "checkpoint_id": AnyStr(),
+            }
+        },
+        subgraph_state_snapshots=None,
+    )
+    assert app.get_state(config, include_subgraph_state=True) == StateSnapshot(
+        values={"my_key": "hi my value"},
+        next=("inner",),
+        config={
+            "configurable": {
+                "thread_id": "1",
+                "checkpoint_ns": "",
+                "checkpoint_id": AnyStr(),
+            }
+        },
+        metadata={
+            "source": "loop",
+            "writes": {"outer_1": {"my_key": "hi my value"}},
+            "step": 1,
+        },
+        created_at=AnyStr(),
+        parent_config={
+            "configurable": {
+                "thread_id": "1",
+                "checkpoint_ns": "",
+                "checkpoint_id": AnyStr(),
+            }
+        },
+        subgraph_state_snapshots={
+            "inner": StateSnapshot(
+                values={"my_key": "hi my value here"},
+                next=(),
+                config={
+                    "configurable": {
+                        "thread_id": "1",
+                        "checkpoint_ns": "inner",
+                        "checkpoint_id": AnyStr(),
+                    }
+                },
+                metadata={
+                    "source": "loop",
+                    "writes": {
+                        "inner_1": {
+                            "my_key": "hi my value here",
+                            "my_other_key": "hi my value",
+                        }
+                    },
+                    "step": 1,
+                },
+                created_at=AnyStr(),
+                parent_config={
+                    "configurable": {
+                        "thread_id": "1",
+                        "checkpoint_ns": "inner",
+                        "checkpoint_id": AnyStr(),
+                    }
+                },
+                subgraph_state_snapshots=None,
+            )
+        },
+    )
+    assert list(app.get_state_history(config, include_subgraph_state=True)) == [
+        StateSnapshot(
+            values={"my_key": "hi my value"},
+            next=("inner",),
+            config={
+                "configurable": {
+                    "thread_id": "1",
+                    "checkpoint_ns": "",
+                    "checkpoint_id": AnyStr(),
+                }
+            },
+            metadata={
+                "source": "loop",
+                "writes": {"outer_1": {"my_key": "hi my value"}},
+                "step": 1,
+            },
+            created_at=AnyStr(),
+            parent_config={
+                "configurable": {
+                    "thread_id": "1",
+                    "checkpoint_ns": "",
+                    "checkpoint_id": AnyStr(),
+                }
+            },
+            subgraph_state_snapshots={
+                "inner": StateSnapshot(
+                    values={"my_key": "hi my value here"},
+                    next=(),
```

Comment:
my only concern is that `next` here is not populated and i think it should. @nfcampos